packages/@react-spectrum/calendar/docs/Calendar.mdx

```diff
@@ -210,3 +210,13 @@ By default, the `Calendar` displays a single month. The `visibleMonths` prop all
   <Calendar aria-label="Event date" visibleMonths={3} />
 </div>
 ```
+
+### Page behavior
```

Comment:
Is there a reason this isn't in DatePicker or DateRangePicker?

Reply:
It wasn't added to `useDatePickerState`/`useDateRangePickerState` in the original PR, but I can include that change in here if we want it.

Reply:
Propagated those down, added them to the types, and added to the docs.